src/hal/drivers/mesa_7i65.comp

```diff
@@ -25,25 +25,25 @@ param rw bit is-bipolar-# [8] = 1 """Set this value to TRUE for a plus/minus
 option extra_setup yes;
 option count_function yes;
 
-variable u32 firstrun;
-
-variable u32 *AD5754_1A;
-variable u32 *AD5754_1B;
-variable u32 *AD5754_1C;
-variable u32 *AD5754_1D;
-variable u32 *AD5754_2A;
-variable u32 *AD5754_2B;
-variable u32 *AD5754_2C;
-variable u32 *AD5754_2D;
-variable u32 *CPLD_write;
-variable u32 *CPLD_read;
-variable u32 *AD7329_write[8];
-variable u32 *AD7329_read[8];
+variable unsigned firstrun;
+
+variable unsigned *AD5754_1A;
+variable unsigned *AD5754_1B;
+variable unsigned *AD5754_1C;
+variable unsigned *AD5754_1D;
+variable unsigned *AD5754_2A;
+variable unsigned *AD5754_2B;
+variable unsigned *AD5754_2C;
+variable unsigned *AD5754_2D;
+variable unsigned *CPLD_write;
+variable unsigned *CPLD_read;
+variable unsigned *AD7329_write[8];
+variable unsigned *AD7329_read[8];
 
 license "GPL";
 author "Andy Pugh / Cliff Blackburn";
 
-include "../../../hal/drivers/mesa-hostmot2/hostmot2.h";
+include "mesa-hostmot2/hostmot2.h";
```

Comment:
This change breaks building on RTAI platforms, with an error like
~~~
/home/buildslave/emc2-buildbot/precise-rtai-x86/precise-i386-realtime-rip/build/src/objects/hal/drivers/mesa_7i65.c:13:36: fatal error: mesa-hostmot2/hostmot2.h: No such file or directory
compilation terminated.
make[4]: *** [/home/buildslave/emc2-buildbot/precise-rtai-x86/precise-i386-realtime-rip/build/src/objects/hal/drivers/mesa_7i65.o] Error 1
make[3]: *** [_module_/home/buildslave/emc2-buildbot/precise-rtai-x86/precise-i386-realtime-rip/build/src] Error 2
~~~~
I'll try to resolve it, but if not we'll have to revert this PR in order to not have a regression on RTAI platforms.

Reply:
agree, on both counts, s16 should have becomt int16_t.

the path change i just something i did to get the comp to build in my env, i have no objection to reverting it.

if the changes are still causing issues then i also agree with reverting the entire pr